lib/rules/no-unexpected-multiline.js

```diff
@@ -24,11 +24,13 @@ module.exports = function(context) {
      */
     function checkForBreakAfter(node, msg) {
         var paren = context.getTokenAfter(node);
+        var beforeParen = node;
         while (paren.value === ")") {
+            beforeParen = paren;
             paren = context.getTokenAfter(paren);
```

Comment:
If i've followed this right, when the `while` is exited
```js
beforeParen = node(")")
paren = node(whatever came next)
```

If this is right, the names should probably be swapped to `paren` and `afterParen`